### PR TITLE
editorial: remove redundancies in API Identifier motivation

### PR DESCRIPTION
**Pull request by tfrauenstein**

### CODE REVIEW

File: chapters/meta-information.adoc [hunk 1]
```diff
@@ -60,16 +60,9 @@ info:
...
-except the API identifier. Based on the API identifier, we can track the API
-life cycle and manage the history and evolution of an API as a sequence of
-API specifications. For instance, we can support to check compatibility and
-correct semantic versioning of an API update against previous API versions
-using the https://github.com/zalando/zally[API Linter Zally].
-
-The API identifier is provided in the `info`-block of the Open API
-specification and must conform to the following specification:
+Each API specification must be provisioned with a globally unique and immutable API identifier. 
+The API identifier is defined in the `info`-block of the Open API
```
Comment (maxim-tschumak) [inline chapters/meta-information.adoc]: I know that we don't always follow this, but the correct spelling is `OpenAPI` (no space between) (https://swagger.io/specification/)

File: chapters/meta-information.adoc [hunk 2]
```diff
@@ -78,20 +71,21 @@ specification and must conform to the following specification:
...
 ----
 
-*Note*: The API identifier must be applied consistently to all created API
-specification versions describing the same API. This explicitly covers all
-applications and deployments as well as compatible and incompatible changes
-of the API that are noticeable addressing the same purpose and consumer group.
-It is responsibility of the API owner to ensure that the application of the
-API identifier together with the semantic API <<116,version>> reflect the
-expected API history.
+API specifications will evolve and every aspect of an OpenAPI specification
```
Comment (maxim-tschumak) [inline chapters/meta-information.adoc]: `every` -> `any`

File: chapters/meta-information.adoc (hunk 1 above)
Comment (tfrauenstein) [reply chapters/meta-information.adoc]: 👍

File: chapters/meta-information.adoc (hunk 2 above)
Comment (tfrauenstein) [reply chapters/meta-information.adoc]: 👍

File: chapters/meta-information.adoc [hunk 3]
```diff
@@ -78,20 +71,21 @@ specification and must conform to the following specification:
...
-It is responsibility of the API owner to ensure that the application of the
-API identifier together with the semantic API <<116,version>> reflect the
-expected API history.
+API specifications will evolve and any aspect of an OpenAPI specification 
+may get changed by API owners. We require API identifiers because we want 
+to support API clients and providers with API lifecycle management features, 
+like change trackability and history or automated backward compatibility checks.  
+The immutable API identifier allows the identification of all API specification 
+versions of an API evolution. And by using  <<116, API semantic version information>> 
+or <<192, API publishing date>> as order criteria you get the "version history"
```
Comment (tkrop) [inline chapters/meta-information.adoc]: Please change `"version history" or "publication history"` to `*version* or *publication history*`. Semantic here is emphasizing not quoting the term.

Comment (tfrauenstein) [reply chapters/meta-information.adoc]: done

**Comment by maxim-tschumak** [comment]
:+1:

File: chapters/meta-information.adoc [hunk 4]
```diff
@@ -78,20 +71,21 @@ specification and must conform to the following specification:
...
-of the API that are noticeable addressing the same purpose and consumer group.
-It is responsibility of the API owner to ensure that the application of the
-API identifier together with the semantic API <<116,version>> reflect the
-expected API history.
+API specifications will evolve and any aspect of an OpenAPI specification 
+may get changed by API owners. We require API identifiers because we want 
+to support API clients and providers with API lifecycle management features, 
+like change trackability and history or automated backward compatibility checks.  
+The immutable API identifier allows the identification of all API specification 
+versions of an API evolution. And by using  <<116, API semantic version information>>
```
Comment (tkrop) [inline chapters/meta-information.adoc]: I would remove `And` - conjunction feels unnecessary here.

File: chapters/meta-information.adoc [hunk 5]
```diff
@@ -78,20 +71,21 @@ specification and must conform to the following specification:
...
 
-*Note*: The API identifier must be applied consistently to all created API
-specification versions describing the same API. This explicitly covers all
-applications and deployments as well as compatible and incompatible changes
-of the API that are noticeable addressing the same purpose and consumer group.
-It is responsibility of the API owner to ensure that the application of the
-API identifier together with the semantic API <<116,version>> reflect the
-expected API history.
+API specifications will evolve and any aspect of an OpenAPI specification 
+may get changed by API owners. We require API identifiers because we want
```
Comment (tkrop) [inline chapters/meta-information.adoc]: May be shorten `may get changed by API owners` -> `may change`, if the who is less relevant?

**Comment by tfrauenstein** [comment]
👍

**Comment by tkrop** [comment]
:+1:

**Pull request merged**